app/controllers/concerns/versionable.rb

```diff
@@ -1,106 +1,123 @@
 module Versionable
-  extend ActiveSupport::Concern
 
-  included do
```

Comment:
`ActiveSupport::Concern` was not being used correctly here. The included block is a callback, and is not necessary to include methods within another class (that's the default behaviour of Modules in Ruby).

Please read the documentation for [ActiveSupport::Concern](https://api.rubyonrails.org/v5.2/classes/ActiveSupport/Concern.html) for tips on correct usage.